ert_shared/storage/extraction.py

```diff
@@ -222,31 +221,39 @@ def _create_observation_transformation(ert, db_observations) -> List[Dict]:
     return [transformation for _, transformation in transformations.items()]
 
 
-@feature_enabled("new-storage")
-def post_update_data(parent_ensemble_id: int, algorithm: str) -> int:
-    server = ServerMonitor.get_instance()
-    ert = ERT.enkf_facade
-
-    observations = requests.get(
-        f"{server.fetch_url()}/ensembles/{parent_ensemble_id}/observations",
-    ).json()
-
-    # create update thingy
-    update_create = dict(
-        observation_transformations=_create_observation_transformation(
-            ert, observations
-        ),
-        ensemble_reference_id=parent_ensemble_id,
-        ensemble_result_id=None,
-        algorithm=algorithm,
-    )
-
-    response = requests.post(
-        f"{server.fetch_url()}/updates",
-        json=update_create,
-    )
-    update = response.json()
-    return update["id"]
+PRIOR_NAME_MAP = {
+    "NORMAL": "normal",
+    "LOGNORMAL": "lognormal",
+    "TRIANGULAR": "trig",
+    "TRUNCATED_NORMAL": "ert_truncnormal",
+    "CONST": "const",
+    "UNIFORM": "uniform",
+    "LOGUNIF": "loguniform",
+    "DUNIF": "ert_duniform",
```

Comment:
Why the ERT prefix?

Reply:
from slack discussion:
The idea  me and zohar discussed was that things that are not following “normal” statistics naming conventions and behaviour are tagged labeled with and ert prefix. In this way ert3 can use an off the shelf statistics library to sample priors, and we are able to separate the different versions 🙂

Reply:
Aight. I'm not sure whether we should diverge or aline on this between `ert2` and `ert3`... As commented on in #1458.

In addition, I guess discrete uniform is a pretty standard distribution no?

Reply:
yes, but not the discrete one that comes from libres, same goes for truncated normal and the error functions. They are not the same as the ones you would find in regular stats libraries

Reply:
Ah, ok. That is a bit interesting 👍🏻

Reply:
> In this way ert3 can use an off the shelf statistics library to sample priors

You should be aware that ERT only uses N(0,1) variables internally, and that distributions like uniform and lognormal and ... are not truly distributions but rather merely *transformations*. If you want to sample arbitrary distributions with an external library you will have to invert the sampled variables back to N(0,1) space - I think that is more difficult than the forward transformation currently utilized.